src/main/java/seedu/manager/command/SortCommand.java

```diff
@@ -0,0 +1,51 @@
+package seedu.manager.command;
+
+import seedu.manager.event.EventList;
+
+/**
```

Comment:
Could an "/@@{author}" tag be added here?

Reply:
but SortCommand is not just for me? it will also be edited by 2 other teammates(by time and by priority).

Reply:
If that's the case then I think it's fine.